src/njoy21/input/CCCCR/Card5/Imat.hpp

```diff
@@ -0,0 +1,11 @@
+struct Imat{
+  using Value_t = int;
+  static std::string name(){ return "imat"; }
+  static std::string description(){
+    return "The imat parameter specifies the numerical isotope identifier\n"
+           "using the ENDF material number.";
+  }
+  static bool verify( const Value_t imat ){
+    return imat > 0;
```

Comment:
This needs to check that imat < 10000.

Reply:
The manual specifies that this value is for ENDF material numbers, which seem to follow the ZZAAA format.  At best, I'd expect this to be limited to < 100000, but we do have elements with Z>100 now, so maybe not?  Hence my lack of an upper-bound.

Reply:
ENDF material numbers (`MAT`) are limited to 4 digits, thus must be < 10000. That is different than the ZA which can have more.